examples/react-template/screens/Hero.tsx

```
@@ -13,66 +11,66 @@ import {
-import { TitleMarkup, TrilogyColor } from "@trilogy-ds/react";
+} from '@trilogy-ds/react/components'
+import React from 'react'
 
 export const HeroScreen = (): JSX.Element => {
   return (
     <Section>
-      <Hero backgroundColor={TrilogyColor.MAIN} overlap inverted>
+      <Hero
+        backgroundColor={TrilogyColor.MAIN}
+        overlap={[
+          <Container>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)